hw/ip/aes/dv/aes_model_dpi/aes_model_dpi.core

```diff
@@ -6,6 +6,9 @@ name: "lowrisc:dv:aes_model_dpi:0.5"
 description: "AES model DPI"
```

Comment:
Why not merge these sources with ../../model?

Reply:
I think even though the DPI calls can go into the C model under `../../model`, they don't belong there. The C model can also be used without the DPI calls. For example, when using the example application for the C model, or when using the Verilator testbench (this calls the C model directly without DPI).